opsdroid/database/mongo/tests/test_mongo.py

```diff
@@ -0,0 +1,77 @@
+import pytest
+
+from opsdroid.database.mockmodules.mongo.mongo_database import DatabaseMongoTest
+from opsdroid.database.mongo import DatabaseMongo
+
+
+@pytest.fixture()
+def database(config):
+    return DatabaseMongo(config)
+
+
+@pytest.mark.parametrize("config", [{"example_item": "test"}])
```

Comment:
I am not particular familar with `pytest.mark.parametrize` since I've just recently started using pytest instead of unittest. But I'm assuming this is loading a fake config, correct? 

Perhaps we could change the "example_item": "test" for "database": "mongo"?

Reply:
Hi, yes `pytest.mark.parametrize` loads fake config. Since "mongo" is DBMS name, wouldn't it be better to have something like "database":"test_db"?

Reply:
Thank you for the explanation that makes sense 😁  yeah that would make more sense then, we are on the mongo tests anyway and you can call your collection whatever you need 👍